### PR TITLE
converted scala starterkit to sbt-project

### PR DESCRIPTION
**Pull request by FloWi**
As discussed previously via mail here's my merge request to convert the scala-starterkit into an sbt project. 

Feel free to reject the PR if you want the folder-structure the way it was before. (source-file in the root-folder and not in src/main/scala). 

See README_SBT.md for details on how to use it.

### CODE REVIEW

File: airesources/Scala/project/Dependencies.scala [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import sbt._
+
+object Dependencies {
+  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.0.3"
```
Comment (Snaar) [inline airesources/Scala/project/Dependencies.scala]: Is this actually used?

File: airesources/Scala/run_game.bat [hunk 2]
```diff
@@ -1,2 +1,3 @@
-call scalac *.scala hlt\*.scala
-halite -d "240 160" "scala MyBot" "scala MyBot"
+sbt assembly
+\halite.exe -d "240 160" "java -jar target\scala-2.12\MyBot.jar" "java -jar target\scala-2.12\MyBot.jar"
```
Comment (Snaar) [inline airesources/Scala/run_game.bat]: Remove \ in front of halite - it's not going to run like that.

File: airesources/Scala/build.sbt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+}
+
+lazy val root = (project in file(".")).
+  settings(
+    inThisBuild(List(
+      scalaVersion := "2.12.3",
+      version := "0.1.0-SNAPSHOT"
+    )),
+    name := "My Bot",
```
Comment (Snaar) [inline airesources/Scala/build.sbt]: Space in "My Bot" looks a bit weird - is this intentional?

File: airesources/Scala/project/Dependencies.scala (hunk 1 above)
Comment (FloWi) [reply airesources/Scala/project/Dependencies.scala]: You're right, it's not used. I removed it.

File: airesources/Scala/build.sbt (hunk 3 above)
Comment (FloWi) [reply airesources/Scala/build.sbt]: Looks weird indeed. Trying to improve it to make it more obvious.

File: airesources/Scala/run_game.bat (hunk 2 above)
Comment (FloWi) [reply airesources/Scala/run_game.bat]: Thanks, haven't used windows for a while.

**Comment by FloWi** [comment]
Ok, thanks for reviewing. Made the changes and amended my previous commit.

**Pull request merged**